reinforcement_learning4e.py

```diff
@@ -310,7 +315,7 @@ def __call__(self, percept):
             self.s = self.a = self.r = None
         else:
             self.s, self.r = s1, r1
-            self.a = argmax(actions_in_state(s1), key=lambda a1: self.f(Q[s1, a1], Nsa[s1, a1]))
+            self.a = max(actions_in_state(s1), key=lambda a1: self.f(Q[s1, a1], Nsa[s1, a1]))
```

Comment:
I have been seeing you replace argmax with max a lot and I want to make sure the replacement is correct, because typically argmax and max return different things (index vs value).

Reply:
Exactly! "np.argmin" and "np.argmax" returns the position of the minimum and maximum value respectively, while the builtins "min" and "max" just the minimum and maximum value respectively.
Bit the different usages of these two functions seem was not clear to anyone because because the role of "min" and "max" was played by "np.argmin" and "np.argmax", in fact in utils.py there was a line of code "argmin = min" and "argmax = max". I've just fixed all the occurrences of argmin and argmax replacing these with just min or max and with "np.argmin" and "np.argmax" when necessary (for example in Viterbi algorithm, learning.py, both were useful).